py/htm_advanced/examples/thing_classification/data/002_master_chef_can.training.log

```diff
@@ -0,0 +1,26 @@
+[(68.1235, -9.7628, 0.7851)/10] => [[5,26,35,68,88,97,328,396,421,466,545,587,608,635,693,732,774,913,964],[91,92,93,94,95,96,97,98,99,100,101,102,103,104,105,106,107,108,109,110,111,112,113,114,115,116,117,118,119,120,121,122,123,124,125,126,127,128,129,130,131]]
```

Comment:
what are these files, their format? Looks like some sensory-motor data for objects? Please add a README in the `/data/`

Reply:
Same comment as for the howtos. I want to tackle the metric classes first.